yapapi/services.py

```diff
@@ -285,15 +286,16 @@ async def start(self):
 
         Additionally, it also assumes that the exe-unit doesn't need any additional parameters
         in its `start()` call (e.g. for the VM runtime, all the required parameters are already
-        passed as part of the agreement between the requestor and the provider).
+        passed as part of the agreement between the requestor and the provider), and parameters
+        passed to `deploy()` are returned by `self.get_deploy_args()` method.
 
         Therefore, this default implementation performs the minimum required for a VM payload to
         start responding to `run` commands. If your service requires any additional operations -
-        you'll need to override this method (possibly starting with a call to `super().start()`)
-        to add appropriate preparatory steps.
+        you'll need to override this method (possibly starting with yielding everything yielded by
+        `super().start()`) to add appropriate preparatory steps.
```

Comment:
```suggestion
        you'll need to override this method (possibly first yielding from the parent - `super().start()` - generator ) 
        to add appropriate preparatory steps.
```

Reply:
iow, "yielding yielded" sounds _very_ awkward

Reply:
To me it's clear, but let's make it your way